src/main/events/HWCheck.ts

```diff
@@ -1,32 +1,359 @@
+import { spawn } from "child_process";
 import HWCheckResponse from "models/HWCheckResponse";
-import { networkInterfaces, hostname } from "os";
+import SoundVolumeViewOutput from "models/SoundVolumeViewOutput";
+import { networkInterfaces, hostname, NetworkInterfaceInfo } from "os";
+import path from "path";
+import ping from "ping";
 
+const SoundVolumeViewPath = process.env.NODE_ENV === 'production'
+    ? path.join(process.resourcesPath, 'assets/SoundVolumeView/SoundVolumeView.exe')
+    : path.join(__dirname, '../../../assets/SoundVolumeView/SoundVolumeView.exe');
 
+const pingConfig = {
+    timeout: 3,
+}
+
+export default async function HWCheck(): Promise<HWCheckResponse> {
+    // The response
```

Comment:
We should probably just bail out early with an error if `process.platform !== 'win32'`, since none of this would be supported on other OSes

I know the carts only run Windows but I (and potentially other devs) might be working on a different OS

Reply:
this is the solution!

Reply:
fixed in 522757e